MAINTAINERS_GUIDELINES.md

```diff
@@ -59,37 +80,65 @@ Please review the following before publishing a new release:
 
 Metadata:
 
-- [ ] update `README.md`
-- [ ] update `CHANGELOG.md`
-- [ ] update version number in `package.json`
+- update `README.md`
+- update `CHANGELOG.md`
+- update version number in `package.json`
+
+Release testing:
+
+- Start, terminate, and monitor ROS core
+    1. launch ROS core monitor with `ros.showMasterStatus`
+    2. start a new ROS core process in background with `ros.startCore`
+    3. check if ROS core monitor shows parameters and nodes correctly
+    4. termintate the ROS core process with `ros.stopCore`
+    5. check if ROS core monitor shows "offline"
+- Create a terminal with ROS environment sourced
+    1. start a new ROS terminal with `ros.createTerminal`
+    2. check if ROS environment variables are properly configured
+- Execute a ROS executable
+    1. start a ROS core process (in another terminal or with `ros.startCore`)
+    2. execute `ros.rosrun` and choose an executable
+    3. check if the executable gets launched
+- Execute a ROS launch file
+    1. execute `ros.roslaunch` and choose a launch file
+    2. check if the launch file gets launched
 
 #### Authorizing a manual release (through the release pipeline)
 
-To authorize a release manually, schedule a release build with the [vscode-ros.ci pipeline][vscode-ros.ci].
+The release process can be automated by triggering a release pipeline from the [vscode-ros.release pipeline][vscode-ros.release]; however, we will be simplifying the process to only release manually for now.
+
+To authorize a release manually, schedule a release build with the proper release branch for the [vscode-ros.release pipeline][vscode-ros.release].
 
 ![schedule a release build][schedule_manual_release_build]
 
-#### Triggering an automatic release (with a new tag)
+After the extension is released, the release branch should be:
+
+1. tagged with version number
```

Comment:
In this case, the git graph will be like:
![image](https://user-images.githubusercontent.com/25241284/59905232-735a2b00-93ba-11e9-9693-aa7319e046f6.png)

Is it the expected result per the steps here? I cannot quickly think of any big issues right now but just check that's the desired tree after this operations.

Reply:
I think this would be fine.